packages/sdk/src/types.ts

```diff
@@ -67,6 +67,11 @@ export enum GRAPHQL_ENDPOINTS  {
   testnet = 'https://interop-testnet.hasura.app/v1/graphql',
 }
 
+export enum FASTNEAR_RPC_ENDPOINTS  {
```

Comment:
Again, we wouldn't need to curate lists here. This also resolves the leakage of our private lava endpoints.